docs/source/1.0/spec/core/shapes.rst

```diff
@@ -388,18 +467,29 @@ targets the ``MyString`` shape in the same namespace.
             }
         }
 
+All shapes that contain members use following ABNF to define members:
+
+.. productionlist:: smithy
+    shape_members           :`empty_shape_members` / `populated_shape_members`
+    empty_shape_members     :"{" `ws` "}"
+    populated_shape_members :"{" `ws` `shape_member_kvp`
+                            :  *(`comma` `shape_member_kvp` `ws`) `trailing_comma` "}"
+    shape_member_kvp        :[`shape_documentation_comments`]
+                            :  `trait_statements`
+                            :  `identifier` `ws` ":" `ws` `shape_id`
+
 
 .. _list:
 
 List
 ====
 
-The :dfn:`list` type represents a homogeneous collection of values. Lists
-are defined using the following grammar:
+The :dfn:`list` type represents a homogeneous collection of values. A list
+statement requires that a :ref:`member <member>` named ``member`` is defined
+in its body. Lists are defined using the following grammar:
 
 .. productionlist:: smithy
-    list_statement          :"list" `list_and_set_body`
-    list_and_set_body       :`identifier` "{" *`trait` "member" ":" `shape_id` [","] "}"
+    list_statement :"list" `ws` `shape_members`
```

Comment:
The production for a `list_statement` no longer includes an identifier to name the `list`: it's not present in this production directly and is [not part of a prefix to the `shape_members` production](https://github.com/awslabs/smithy/pull/434/files#diff-686d75bfe1ffdd42ae4f485cb47b8000R473). It appears this issue was carried forward to many other `shape_members` related shape productions.